README.md

```diff
@@ -1,31 +1,34 @@
 ### Instructions
 
-## Updating a git repo to a new tag 
+## TopTagger Code
 
-Commit all outstanding change (if any) to a new branch.
-```
-git checkout -b NEW_BRANCH_NAME
-```
+To Checkout TopTagger Code:
 
-Fetch the remote changes and check out the new tag 
-```
-git fetch
-git checkout NEW_TAG_NAME
 ```
+cmsrel CMSSW_9_4_4
+cd CMSSW_9_4_4
+cmsenv
 
-If necessary merge your outstanding changes back into the working branch , note that there may be merge conflicts to resolve
-```
-git checkout -b ANOTHER_NEW_BRANCH_NAME
-git merge NEW_BRANCH_NAME
+## Checkout Tagtagger
+cd $CMSSW_BASE/src
+git clone git@github.com:susy2015/TopTagger.git
+cd TopTagger/TopTagger/test/
+./configure OPENCVDIR=/uscms_data/d3/pastika/zinv/dev/CMSSW_7_4_8/src/opencv/
+make -j 8 
 ```
 
+## SUSYAnaTools Code
+
 You can then compile the SUSYAnaTools
 ```
+cd $CMSSW_BASE/src
+git clone -b AddNewTagger git@github.com:susy2015/SusyAnaTools.git
+
 cd $CMSSW_BASE/src/SusyAnaTools/Tools/
 autoconf
 #note if you are not on the LPC you will have to follow the instructions below to install OpenCV on your system
```

Comment:
There are no instructions below

Reply:
Below these line are 
```./configure OPENCVDIR=/uscms_data/d3/pastika/zinv/dev/CMSSW_7_4_8/src/opencv/
make -j 8 ```
That is the instruction for compile

Reply:
I am referring to "instructions below to install OpenCV on your system"

Reply:
I see. Let me add that